tagEquationWrapper.js

```diff
@@ -0,0 +1,103 @@
+/*
+LANDWAR Unit Builder - V0.0.1
+author: Peter Roohr
+overview
+    + wraps Tag Equation functions
+    + wrapper only needs unit object prescribed in UnitEquations.js
+*/
+
+function tagCall_ArmorPiercing(unitData){ 
```

Comment:
I don't like this extra wrapper around every function. This is called WET (we enjoy typing)

Reply:
removed this code, we're all set.